LayoutTests/http/tests/paymentrequest/ApplePayModifier-disbursementPaymentRequest.https.html

```diff
@@ -1,116 +1,187 @@
-<!doctype html>
+<!DOCTYPE html>
 <meta charset="utf-8" />
 <title>
-  Tests for providing `disbursementPaymentRequest` as part of
-  `ApplePayModifier`.
+    Tests for providing `disbursementPaymentRequest` as part of
+    `ApplePayModifier`.
 </title>
 <script src="/js-test-resources/ui-helper.js"></script>
 <script src="/resources/payment-request.js"></script>
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
+
 <body>
-  <script>
-    setup({ explicit_done: true, explicit_timeout: true });
-
-    test((test) => {
-      const method = validPaymentMethod();
-      method.data.features = ["paymentRequestDisbursements"];
-      new PaymentRequest([method], validPaymentDetails());
-    }, "Should have a feature for `disbursementPaymentRequest`.");
-
-    const RESOLVED_PROMISE = Promise.resolve({});
-
-    const lineItems = {
-      total: {
-        label: "Chime",
-        amount: "15.00",
-      },
-      ifo: {
-        label: "Instant Transfer Fee",
-        amount: "0.15",
-        disbursementLineItemType: "instantFundsOutFee",
-      },
-      disbursement: {
-        label: "Total Amount",
-        amount: "14.85",
-        disbursementLineItemType: "disbursement",
-      },
-    };
-
-    const validLineItems = [
-      lineItems.total,
-      lineItems.ifo,
-      lineItems.disbursement,
-    ];
-
-    function modifiersWithData(data, lineItems) {
-      data.additionalLineItems = lineItems;
-      return [
-        { supportedMethods: "https://apple.com/apple-pay", data: { ...data } },
-      ];
-    }
-
-    function testValidDisbursementPaymentRequest(description, options) {
-      const {
-        initialDisbursementPaymentRequest,
-        expectedDisbursementPaymentRequest,
-      } = options;
-      user_activation_test(async (test) => {
-        const detailsInit = {
-          total: {
-            label: "Disbursement",
-            amount: {
-              value: "14.85",
-              currency: "USD",
+    <script>
+        setup({ explicit_done: true, explicit_timeout: true });
+
+        test((test) => {
+            const method = validPaymentMethod();
+            method.data.features = ["paymentRequestDisbursements"];
+            new PaymentRequest([method], validPaymentDetails());
```

Comment:
I wonder if we should assure this with something like:

```suggestion
            try {
               method.data.features = ["not a thing"];
               new PaymentRequest([method], validPaymentDetails());
               assert_unreached("Should have thrown")
            } catch {
               method.data.features = ["paymentRequestDisbursements"];
               new PaymentRequest([method], validPaymentDetails());              
            }
            
```

Reply:
Not opposed to it - done!